### PR TITLE
Hardcode Supporter Plus pricing and threshold amounts

### PR DESCRIPTION
**Pull request by marialani**
## What are you doing in this PR?
This hardcodes the pricing and threshold amounts on the front-end for the Supporter Plus checkout. 

[**Trello Card**](https://trello.com/c/r56jhHIM/600-set-all-pricing-and-threshold-amounts-for-the-new-supporter-plus-product-checkouts)

### CODE REVIEW

**Comment by marialani** [comment]
[Set all pricing and threshold amounts for the new Supporter Plus product checkouts](https://trello.com/c/r56jhHIM/600-set-all-pricing-and-threshold-amounts-for-the-new-supporter-plus-product-checkouts)

File: support-frontend/assets/helpers/redux/checkout/product/supporterPlus/supporterPlusAmounts.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+			amounts: [25, 50, 100, 250],
+			defaultAmount: 50,
+		},
+		MONTHLY: {
+			amounts: [8, 10, 18, 22],
+			defaultAmount: 10,
+		},
+		ANNUAL: {
+			amounts: [50, 95, 250, 500],
+			defaultAmount: 0,
```
Comment (GHaberis) [inline support-frontend/assets/helpers/redux/checkout/product/supporterPlus/supporterPlusAmounts.ts:59]: is 0 the correct `defaultAmount` here?

File: support-frontend/assets/helpers/redux/checkout/product/supporterPlus/supporterPlusAmounts.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+			amounts: [50, 100, 250, 500],
+			defaultAmount: 100,
+		},
+		MONTHLY: {
+			amounts: [5, 13, 20, 50],
+			defaultAmount: 13,
+		},
+		ANNUAL: {
+			amounts: [60, 120, 199, 500],
+			defaultAmount: 0,
```
Comment (GHaberis) [inline support-frontend/assets/helpers/redux/checkout/product/supporterPlus/supporterPlusAmounts.ts:87]: is 0 the correct `defaultAmount` here?

Comment (marialani) [reply support-frontend/assets/helpers/redux/checkout/product/supporterPlus/supporterPlusAmounts.ts:87]: No this one needs updating

**Review by GHaberis — APPROVED** [review]
Looks good!

**Pull request merged**